src/actions/whenanyaction.ts

```diff
@@ -0,0 +1,12 @@
+import { ActionType, IAction } from "../classes";
+import { ProperTask } from "../tasks/internalTasks";
+
+/** @hidden */
+export class WhenAnyAction implements IAction {
+    public readonly actionType: ActionType = ActionType.WhenAll;
```

Comment:
Should this be `ActionType.WhenAny`?

Reply:
I think you're right. Huh, I wonder why no tests caught this. I'll look into this, thanks!